### PR TITLE
add pics and placeholders for new starters

### PR DESCRIPTION
**Pull request by catherine-mediasuite**
- have added images for exisiting placeholders
- added placeholders for new starters
- have added bios

note: 
- awaiting approved bios from Tim O.

### CODE REVIEW

File: src/people/index.html [hunk 1]
```diff
@@ -258,11 +258,52 @@ <h3 class="person-name-hover">Brad</h3>
       </div>
       <!-- .person-wrap -->
 
+      <!-- Callum -->
+      <div id="person-tim-o" class="person-wrap js-iso-element">
```
Comment (jooleearr) [inline src/people/index.html:262]: Just need to update the id to be unique here. I have this nagging feeling that they're used for something but don't quite remember what 😬  Blame Julia from 6 years ago about how mystery meat this whole site is now 😂

File: src/people/index.html [hunk 2]
```diff
@@ -898,6 +980,47 @@ <h3 class="person-name-hover">Kirsty</h3>
...
+                    <a href="https://nz.linkedin.com/in/marjolein-visser-1b8b3522" class="linkedin-link" target="_blank">
+                      <img src="{{ "/img/linkedin-logo.png " | prepend: site.baseurl }}" alt="Linkedin logo" class="linkedin-img">
+                    </a>
+                  </span>
+                  <!-- <div class="js-image-fill bio-real-image">
+                    <img src="{{ "/img/people/real-mar.jpg" | prepend: site.baseurl }}" alt="Marjolein Visser" />
+                  </div> -->
+                </div>
+                <div class="bio-body">
+                  <p>Rembrandt and van Gogh have nothing on Media Suite’s very own Dutch master - at least when it comes to user interface and user experience design. Mar hails from Alkmaar in the Netherlands, where she developed a love of cycling, art house movies, and digital design. Creating smooth, intuitive user experiences is Mar’s MO, especially when it’s for projects with real world impact.</p>
```
Comment (jooleearr) [inline src/people/index.html:1011]: TIL the 'v' in van Gogh is lowercase :)

File: src/people/index.html [hunk 3]
```diff
@@ -1508,6 +1631,88 @@ <h3 class="person-name-hover">Subhra</h3>
...
+                  <p>Tim has been working with subject matter experts to demystify challenging topics for almost 20 years - as a motoring journalist, content developer, technical writer, and marketer. He hasn’t got the usual CV, but he has a breadth of experience championing end users that runs through all his work. Hailing from the UK, Tim is an avid follower of Formula One and a big fan of English bitter served at room temperature.</p>
+                  <!-- <div class="fancy-pants-font js-blog-link" data-blog-link="/author/tim-grey"></div> -->
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- .bio-inner -->
+        </div>
+        <!-- .bio-wrap -->
+        <h3 class="person-name-hover">Tim G.</h3>
```
Comment (jooleearr) [inline src/people/index.html:1671]: I don't think we've added the '.' after the last name initial elsewhere. This is probably more correct tbh but maybe just update the others with a surname initial to be consistent #nitpicking 😅

**Review by jooleearr — CHANGES_REQUESTED** [review]
Hiya - Rae is keen to get this out as is and just add the missing bios when they finally land. Just a couple of super minor/annoying changes but then I think you're good to merge to master (which should trigger a deploy) :)

File: src/people/index.html (hunk 3 above)
Comment (catherine-mediasuite) [reply src/people/index.html:1671]: Added fullstops

**Pull request merged**